### PR TITLE
v1.7.1

### PR DESCRIPTION
**Pull request by idinium96**
**Fixes**
- #130 - fix the Autokeys Scrap Adjustment feature.
- fix bot list everything to high-value items.
- fix "Missing conversion rate for keys in refined" because of bad key prices from sbn.tf - the key prices contains keys in it. This is fixed by completely remove sbn.tf as the source of the key prices.

**Adds**
- add an option to not disable any accepted high-value items.
	- new variable: `DISABLE_HIGH_VALUE_HOLD`

**Changes/Updates**
- cleanup `uptime` logic.
- update `bptf-listing-2` module, **buy orders** for targeted items (Crates, Unusualifier, Strangifier, Killstreak Kit, Killstreak Kit Fabricator and Chemistry Set) are now supported - but not painted items, skins, war paint, and items with craft number.
- #131 - Do not change key entry if Autokeys enabled.
- #124 - new bot reply when adding/updating item prices.

### CODE REVIEW

**Comment by idinium96** [comment]
No, this does not work.

File: src/classes/MyHandler.ts [hunk 1]
```diff
@@ -2028,17 +2026,21 @@ export = class MyHandler extends Handler {
...
+            // If the bot has been up for ~1 day, show the exact amount of hours
+            // If the bot has been up for ~1 month, show the exact amount of days
+            // Otherwise, show the uptime as it is
+            if (hoursDiff >= 21.5 && hoursDiff < 35.5) {
+                log.debug(`Bot has been up for a day (${hoursDiff} hours).`);
+            } else if (daysDiff >= 25.5) {
+                log.debug(`Bot has been up for a month (${daysDiff} days).`);
+            } else {
+                log.debug(`Bot has been up for ${uptimeAsMoment.from(currentTime, true)}.`);
+            }
```
Comment (Bonfire) [inline src/classes/MyHandler.ts:2043]: These changes may also need to be added to the `!uptime` command

**Review by Bonfire — APPROVED** [review]
PR looks good. I may modify the uptime logic for the `!uptime` command

**Comment by idinium96** [comment]
> PR looks good. I may modify the uptime logic for the `!uptime` command

Thanks. PR merged.

**Review by Bonfire — APPROVED** [review]

**Pull request merged**